src/components/Input/index.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react"
+import PropTypes from "prop-types"
+import { InputCls } from "./styles"
+import { cx } from "emotion"
+
+Input.propTypes = {
+  className: PropTypes.string,
+  onChange: PropTypes.func,
+  label: PropTypes.string,
+  value: PropTypes.string,
+  disabled: PropTypes.bool,
+  errorMessage: PropTypes.string
+}
+
+function Input({ className, onChange, label, value, disabled, errorMessage }) {
+  const [inputValue, setInputValue] = useState("")
+  const [isDisabled, setIsDisabled] = useState(false)
+  const [error, setError] = useState(false)
+
+  errorMessage = "We are sorry, an error occured. Please try again."
```

Comment:
This should be removed. If you want to test the component add to the TestPage and pass the errorMessage property to the input

Reply:
you mean like :
`<Input errorMessage={errorMessage}/>`  ?